tests/test_minimize.py

```diff
@@ -169,8 +169,9 @@ def test_basinhopping(self):
         res = basinhopping(func, x0, minimizer_kwargs={"method": "BFGS"}, niter=200)
         np.random.seed(555)
         x, = parameters('x')
-        fit = BasinHopping(func, [x])
-        fit_result = fit.execute(minimizer_kwargs={"method": "BFGS", 'jac': False}, niter=200)
+        fit = BasinHopping(func, [x], local_minimizer=BFGS)
```

Comment:
```
fit = Fit(func, minimizer=BasinHopping)
fit.execute(local_minimizer=BFGS)
```

Reply:
This is meant as tests on the Minimizer directly, not if it plays nice with Fit. That's a couple of lines down ;).